### PR TITLE
dashboard

### PR DESCRIPTION
**Pull request by giovamarr**
bootstrap: dashboard, footer, header y navbar

### CODE REVIEW

**Comment by oberruti** [comment]
@giovamarr As a first view, don’t commit the sql lite file, please delete from here! Thanks

**Comment by oberruti** [comment]
Can you post a GIF or screenshots always that you change something in the frontend? This is really helpful for the reviewer! Thanks.

File: practico_08/autogestion_alumnos/__init__.py [hunk 1]
```diff
@@ -14,6 +15,7 @@ def create_app(config_class):
     app = Flask(__name__)
     app.config.from_object(config_class)
 
+    bootstrap = Bootstrap(app)
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/__init__.py:18]: Is it more important that the `init` ? I think that this should be the last in the list of these initializations. Can you change this?

File: practico_08/autogestion_alumnos/templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<footer class="py-4 bg-light mt-auto">
+    <div class="container-fluid">
+        <div class="d-flex align-items-center justify-content-between small">
+            <div class="text-muted">Copyright &copy; Your Website 2020</div>
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/templates/footer.html]: I think that you forget to change something here, please review it.

**Review by oberruti — CHANGES_REQUESTED** [review]
Good changes, I will wait for a gif or images to see what you did here. Also, did you add the `.sql` file in the `.gitignore`? Thanks.

**Comment by giovamarr** [comment]
soporte_dashboard
Dashboard ready

File: practico_08/autogestion_alumnos/templates/base.html [hunk 3]
```diff
@@ -15,5 +15,7 @@
         {% include 'navbar.html' %}
     {% endblock %}
 
-    {% block content %}{% endblock %}
+    {% block content %}
+    {% endblock %}
+    {% include 'footer.html' %}
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/templates/base.html]: I think this is wrong, the include shouldn't be between block and endblock? if not we should delete these two first lines.

File: practico_08/autogestion_alumnos/templates/dashboard.html [hunk 4]
```diff
@@ -171,8 +147,8 @@ <h4 class="card-title">
         </div>
       </div>
 
-  </div>
-    {% include 'footer.html' %}
-            </div>
-        </div>
+    </div>
+    </div>
+
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/templates/dashboard.html:82]: please delete this empty line

File: practico_08/autogestion_alumnos/templates/dashboard.html [hunk 5]
```diff
@@ -1,154 +1,103 @@
-{% extends 'base.html' %}
-
-{% block title %}
-    {{ super() }}
-    Home
-{% endblock %}
-
-{% block content %}
-<main>
+<!DOCTYPE html>
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/templates/dashboard.html:1]: I don't know where to write this so I'm writing it here, PLEASE PLEASE, WRITE RIGHT NAMES, images/1.png is not a good name, please, change all those image names, to something like logo-calendar or similar. Thanks!

**Review by oberruti — CHANGES_REQUESTED** [review]
Some changes needed. Really really good job Giova!

**Comment by giovamarr** [comment]
soporte_tasks
Fix and template tasks

**Pull request merged**